source/Octopus.Manager.Tentacle/TentacleConfiguration/TentacleManager/TentacleManagerModel.cs

```diff
@@ -166,7 +167,8 @@ void Reload(ApplicationInstanceRecord applicationInstance)
             }
             else
             {
-                var listens = tencon.TrustedOctopusServers.Where(t => t.CommunicationStyle == CommunicationStyle.TentaclePassive).ToList();
+                var listens = tencon.TrustedOctopusServers.Where(t => t.CommunicationStyle == CommunicationStyle.TentaclePassive ||
+                    (t.CommunicationStyle == CommunicationStyle.KubernetesTentacle && t.AgentCommunicationMode == TentacleCommunicationModeResource.Listening)).ToList();
```

Comment:
Looks like a rogue `AgentCommunicationMode` property still floating around?

Reply:
Ah damn, good catch. Yep I'll fix that up 👍